src/Pages/Reconstruction.jsx

```diff
@@ -63,8 +66,30 @@ export class Reconstruction extends Component {
   submitHandler(e) {
     e.preventDefault();
     const myData = document.getElementById('form');
-    let formData = new FormData(myData);
-    console.log(formData);
+
+    this.setState({
+      loading: true,
+    });
+
+    const url =
+      'https://threedify-api-staging.herokuapp.com/reconstructions/create';
+
+    axios
+      .post(url, myData)
```

Comment:
The data here need to be a `FormData` object not the `HTMLFormElement`.
```suggestion
      .post(url, new FormData(myData))
```